x-pack/plugins/lists/server/routes/read_privileges_route.mock.ts

```diff
@@ -88,7 +88,7 @@ export const getReadPrivilegeMock = (
       manage_index_templates: booleanValues,
       manage_ingest_pipelines: booleanValues,
       manage_ml: booleanValues,
-      manage_own_api_key: false,
+      manage_own_api_key: booleanValues,
```

Comment:
This value was updated so now the `elastic` user with `all` privileges also has the `manage_own_api_key: true`.